sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/MethodOverrideAddsParams.Fixed.cs

```diff
@@ -38,4 +38,18 @@ abstract class Derived : Base
         {
         }
     }
+
+    public interface IFirst
+    {
+        public virtual void Method(int[] numbers)
+        {
+        }
+    }
+
+    public class Second : IFirst
+    {
+        public void Method(params int[] numbers) // Compliant - The default interface method can be accessed only after cast.
```

Comment:
do you mean the `base` interface? (why default?)

Reply:
I was referring to the language feature `Default Interface Methods` but indeed this can be a bit confusing.